pages/web/microsoft-collaborative-solutions/exchange_adding_domain/guide.it-it.md

```diff
@@ -1,117 +1,155 @@
+excerpt: 'Dowiedz się, jak dodać domenę do Twojej usługi Exchange'
+section: 'Konfiguracja usługi Exchange'
 ---
 
-**Ultimo aggiornamento: 28/01/2019**
+> [!primary]
+> Questa traduzione è stata generata automaticamente dal nostro partner SYSTRAN. I contenuti potrebbero presentare imprecisioni, ad esempio la nomenclatura dei pulsanti o alcuni dettagli tecnici. In caso di dubbi consigliamo di fare riferimento alla versione inglese o francese della guida. Per aiutarci a migliorare questa traduzione, utilizza il pulsante "Modifica" di questa pagina.
+>
+
+**Ultimo aggiornamento: 09/09/2022**
 
 ## Obiettivo
 
-Aggiungere uno o più domini su Exchange è un’operazione fondamentale per utilizzare gli account inclusi in questa soluzione.
+Aggiungere un dominio su Exchange è un’operazione fondamentale per utilizzare gli account inclusi in questa soluzione. È anche possibile aggiungere più domini a un servizio Exchange o Email Pro.
 
-**Questa guida ti mostra come aggiungere un dominio al tuo servizio Exchange.**
+**Questa guida ti mostra come aggiungere un dominio alla piattaforma Exchange o Email Pro.**
 
 ## Prerequisiti
 
-- Disporre di una [soluzione Exchange](https://www.ovhcloud.com/it/emails/){.external}
+- Disporre di una soluzione [Exchange](https://www.ovhcloud.com/it/emails/) o [Email Pro](https://www.ovhcloud.com/it/emails/email-pro/))
 - Aver registrato uno o più domini
-- Poter modificare la configurazione del dominio (zona DNS)
-- Avere accesso allo [Spazio Cliente OVH](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.it/&ovhSubsidiary=it){.external}
+- Poter modificare la configurazione del dominio ([zona DNS](https://docs.ovh.com/it/domains/web_hosting_modifica_la_tua_zona_dns/))
+- Avere accesso allo [Spazio Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.it/&ovhSubsidiary=it)
 
 ## Procedura
 
-### Step 1: accedi alla gestione del servizio
+### Accedi alla gestione del tuo servizio
 
-Una volta che il tuo Exchange è stato creato e risulta disponibile, è possibile gestirlo dallo [Spazio Cliente OVH](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.it/&ovhSubsidiary=it){.external}:
+Una volta che il tuo Exchange o Email Pro è stato creato e reso disponibile, è possibile gestirlo dallo [Spazio Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.it/&ovhSubsidiary=it).
 
- Clicca su `Microsoft`{.action} > `Exchange`{.action} e seleziona il tuo servizio.
+Nella sezione `Web Cloud`{.action}:
 
-> [!primary]
->
-> Nello Spazio Cliente OVH il nome dei servizi Exchange inizia con **hosted-** o **private-**, seguito da una parte del tuo identificativo cliente e una cifra (1 per il primo servizio installato, 2 per il secondo, ecc...).
->
+- **Exchange**: Clicca su `Microsoft`{.action} e poi su `Exchange`{.action}. 
+- **Email Pro**: Clicca su `Email Pro`{.action}.
 
-### Step 2: aggiungi un dominio
+e seleziona il nome del servizio.
```

Comment:
some part is missing here

Reply:
this part is like FR version, nothing is missing